core/descriptors/src/org/jetbrains/kotlin/renderer/DescriptorRendererImpl.kt

```diff
@@ -630,6 +634,8 @@ internal class DescriptorRendererImpl(
 
                 if (includeAdditionalModifiers) {
                     renderAdditionalModifiers(function, builder)
+                } else if (includeSuspendModifier) {
```

Comment:
Why else?

Reply:
`includeAdditionalModifiers` implies `includeSuspendModifier`. If I left out the `else`, `suspend` would be rendered twice.